v2/tools/generator/internal/jsonast/jsonast.go

```diff
@@ -8,6 +8,7 @@ package jsonast
 import (
 	"context"
 	"fmt"
+	"github.com/Azure/azure-service-operator/v2/tools/generator/internal/astbuilder"
```

Comment:
minor: import ordering, put this at the bottom w/ the other `aso` imports.

Reply:
Fixed.